master/buildbot/util/__init__.py

```diff
@@ -209,6 +209,35 @@ def datetime2epoch(dt):
         return calendar.timegm(dt.utctimetuple())
 
 
+# TODO: maybe "merge" with formatInterval?
+def human_readable_delta(start, end):
```

Comment:
I think this would belong to reporter.util as I dont see how it could be useful appart from that.